osm-seed/values.yaml

```diff
@@ -864,3 +865,21 @@ planetFiles:
   image:
     name: ""
     tag: ""
+# ====================================================================================================
+# Cgimap
+# ====================================================================================================
+cgimap:
+  enabled: false
+  image:
+    name: ''
+    tag: ''
+  resources:
+    enabled: false
+    requests:
+      memory: '20Gi'
```

Comment:
I know this likely gets over-ridden, but that seems very high as a default? Can we keep the defaults low and over-ride as needed?

Reply:
yes, you are right, i will update that!